apps/antalmanac/src/components/Header/LoadSaveFunctionality.tsx

```diff
@@ -32,6 +33,17 @@ interface LoadSaveButtonBaseState {
     rememberMe: boolean;
 }
 
+class SaveLoadIcon extends PureComponent<{ loading: boolean; actionName: 'Save' | 'Load' }> {
+    render() {
+        const { loading, actionName } = this.props;
+        if (loading) {
+            return <CircularProgress size={20} color={'inherit'} />;
```

Comment:
If it's a string prop, you don't need extra braces around it.

```suggestion
            return <CircularProgress size={20} color='inherit' />;
```